client/src/components/pages/faq/FAQ.tsx

```diff
@@ -125,9 +127,35 @@ const FAQ = () => {
   const faqContent =
     userType === "candidate" ? faqDataCandidates : faqDataCompanies;
 
+  useEffect(() => {
+    if (isTTSActive) {
+      window.speechSynthesis.cancel(); // Var olan TTS okumalarını iptal et
+
+      const msg = new SpeechSynthesisUtterance(selectedText);
+      window.speechSynthesis.speak(msg); // Seçilen metni sesli olarak oku
+    }
+  }, [selectedText, isTTSActive]);
+
+  const handleTTSButtonClick = () => {
+    setTTSActive(!isTTSActive);
+  };
+
+  const toggleTTS = () => {
```

Comment:
1. why this file is being modified?
2. why is being modified in this PR?
3. from which design the textTopSpeech part is this following?
4. why there are comments not in english?